### PR TITLE
Fixed the buggy maxPlayers data member

### PR DESCRIPTION
**Pull request by groadybones**
Fixes - fixed before the bug report could even exist

#### Overview of changes

-no more problems with max players when switching between game modes
-players no longer have to re-enter names that were removed by switching game modes, the names are stored and re-added as a player character
-still only two player games

Reviewer: @johnanvik

### CODE REVIEW

File: src/components/SharedComponents/SettingsComponent.vue [hunk 1]
```diff
@@ -24,18 +24,35 @@
...
+          <div class="container gameTypes">
+            <div class="col-md-2">
+              <input type="radio" id="hotseat" name="gameType" value="hotseat" @click="checkStoredPlayers()">
+              <label for="hotseat"><b>Hotseat: </b><br>Add two player names for local multiplayer</label>
+            </div>
+            <div class="col-md-2">
+              <input type="radio" id="oneAI" name="gameType" value="oneAI" @click="addAI(1)">
+              <label for="oneAI"><b>One bot: </b><br>Face off against an AI opponent in a 1v1 grudge match</label>
+            </div>
+            <!-- The div below is commented out because its the only version that still causes issues and I want to do a pull request for a working version tonight. Will try to fix tomorrow morning -->
```
Comment (johnanvik) [inline src/components/SharedComponents/SettingsComponent.vue]: Remove comment

File: src/components/SharedComponents/SettingsComponent.vue [hunk 2]
```diff
@@ -95,16 +105,16 @@
         dataToggle: false,
         modalTitle: 'Welcome to a Program Wars!',
         localPlayers: [{name: '', isAi: false}],
+        storedPlayers: [],
         newPlayer: '',
         gameStart: false,
-        selected: '35',
+        selected: '50',
```
Comment (johnanvik) [inline src/components/SharedComponents/SettingsComponent.vue]: Based on what I was seeing with kids this morning, 50 might make the game too long. Let's leave it at 35.

**Comment by groadybones** [comment]
Okay, I think that could've been because it went back to two playing fields that aren't consistently active. I think with the concept rework for grouping, we'll want to set it back to 50 cause the games do speed up quite a bit. However, for now I'll switch it back to 35.

File: src/components/SharedComponents/SettingsComponent.vue (hunk 1 above)
Comment (johnanvik) [reply src/components/SharedComponents/SettingsComponent.vue]: Done

**Pull request merged**